src/deadline/cinema4d_adaptor/Cinema4DAdaptor/adaptor.py

```diff
@@ -272,7 +275,9 @@ def _start_cinema4d_client(self) -> None:
         # XXX: on linux we need to run the c4d env setup script first, this env
         # var allows us to use a wrapper around Commandline. Ideally a conda env
         # does this for us.
-        c4d_exe_env = os.getenv("DEADLINE_CINEMA4D_EXE", "")
+        # On Linux this should be a path similar to this: /opt/maxon/cinema4dr2024.200/bin/Commandline
+        # On Windows it should be a path similar to this: "C:\Program Files\Maxon Cinema 4D R26\Commandline.exe"
+        c4d_exe_env = os.environ.get("CINEMA4D_ADAPTOR_CINEMA4D_EXE", "")
```

Comment:
Thanks for adding in the comment here, it definitely helps!

Since we're specifically using `commandline.exe` instead of c4d or its ilk, this should probably be: `CINEMA4D_ADAPTOR_COMMANDLINE_EXE`.